dev-util/radare2/radare2-2.4.0.recipe

```diff
@@ -4,35 +4,42 @@ able to open disk files, analyzing binaries in several common and less common \
 formats, disassemble code for several CPU architectures, debug programs, attach \
 to remote GDB servers, and much more."
 HOMEPAGE="https://radare.org/"
-COPYRIGHT="2009-2017 nibble, pancake"
+COPYRIGHT="2009-2018 nibble, pancake"
```

Comment:
There are more holders than just those two. Please update `COPYRIGHT` with this [list] from Debian

[list]: http://metadata.ftp-master.debian.org/changelogs/main/r/radare2/radare2_2.4.0+dfsg-1_copyright